src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -138,6 +130,22 @@ AutomatedRssDownloader::~AutomatedRssDownloader()
     delete m_episodeRegex;
 }
 
+void AutomatedRssDownloader::connectRuleFeedSlots()
+{
+    qDebug() << Q_FUNC_INFO << "Connecting rule and feed slots";
+    connect(ui->listRules, SIGNAL(itemSelectionChanged()), this, SLOT(updateRuleDefinitionBox()), Qt::UniqueConnection);
+    connect(ui->listRules, SIGNAL(itemChanged(QListWidgetItem *)), this, SLOT(handleRuleCheckStateChange(QListWidgetItem *)), Qt::UniqueConnection);
+    connect(ui->listFeeds, SIGNAL(itemChanged(QListWidgetItem *)), this, SLOT(handleFeedCheckStateChange(QListWidgetItem *)), Qt::UniqueConnection);
+}
```

Comment:
Looking closer, `Qt::AutoConnection = 0` so no need to `|` - it's implicit if one of the other connection type flags isn't set.